### ISSUE TITLE
Significantly revamp documentation

### ISSUE THREAD

**Issue by OfficerHalf** [OfficerHalf/obsidian-auto-class#33]
Documentation is missing:

- Usecases. Why would you use the plugin if you don't know what it actually does?
- Examples with CSS snippets to test with
- Guide on converting a theme to a usable snippet
- Up to date images
- global scope support
